pkg/koordlet/runtimehooks/hooks/groupidentity/interceptor.go

```diff
@@ -35,6 +35,10 @@ func (b *bvtPlugin) SetPodBvtValue(p protocol.HooksProtocol) error {
 		klog.V(5).Infof("hook plugin rule is nil, nothing to do for plugin %v", name)
 		return nil
 	}
+	if !b.initialized() {
```

Comment:
if !rule.enabled && (/proc/sys/enabled exist) {
   // /proc/sys/enabled will be zero by rule module
   // do nothing since no bvt file 
   return
}

if rule.enable {
    try_init(/proc/sys/enabled=1)
}

Reply:
done